client-libraries/python/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[project]
+name = "modality"
+version = "0.0.1"
+authors = [
+  { name="Russell Mull", email="russell@auxon.io" },
+]
+description = "A python client library for the Modality API"
+readme = "README.md"
+requires-python = ">=3.7"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: Apache License",
+    "Operating System :: OS Independent",
+]
```

Comment:
To get this to install I needed to `s/Apache License/Apache Software License`. 

ref: https://pypi.org/pypi?%3Aaction=list_classifiers